source/operations/concepts/healing.rst

```diff
@@ -0,0 +1,110 @@
+.. _minio-concepts-healing:
+
+==============
+Object Healing
+==============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+What is healing?
+----------------
+
+Healing is MinIO's ability to restore an object that has been damaged, corrupted, or partially lost.
+The loss can come from random corruptions in the form of 
+- :ref:`bit rot <minio-ec-bitrot>`
+- the loss of one or more drives due to failure or outages
+- the corruption or loss of one or more shards of an object for other reasons
+
+Healing and Erasure Coding
+--------------------------
+
+The ability of MinIO to restore a damaged object relates directly to the following:
+
+- total number of drives in the :ref:`erasure set <minio-erasure-coding>` where the object exists
+- :ref:`parity setting <minio-ec-parity>` for the erasure set
+
+  Parity refers to the number of extra shards MinIO create when writing the object.
+  For example, an erasure set may have eight total drives and use three drives during a write for parity.
+  In that scenario, MinIO splits the object into five parts written to five of the drives.
+  MinIO uses the remaining three drives in the erasure set for parity shards.
```

Comment:
This *might* imply that we always use the same drives for parity or data. It's randomized, so maybe

```suggestion
For example, MinIO splits an object into 5 data and 3 parity shards and distributes those shards across 8 drives.
```

Reply:
🤔  Interesting thought. Taking your suggestion for the most part.